src/disclosures/css/disclosures.less

```diff
@@ -243,21 +243,47 @@
   &_content {
 
     .respond-to-min(@bp-sm-min, {
-      .grid_column(9);
+      .grid_column(6);
     });
+  }
 
-    .respond-to-min(@bp-lg-min, {
-      .grid_column(7);
-    });
+  &_subheading {
+    .heading-4();
+  }
 
-    .respond-to-min(@bp-xl-min, {
+  &_about-wrapper {
```

Comment:
This was causing some weirdness for me, especially between `bp-sm-min` and `bp-med-min`. I think this should give the right layout:

```less
  &_about-wrapper {
    padding-top: unit(20px / @base-font-size-px, em);
    padding-bottom: unit(20px / @base-font-size-px, em);
    margin-top: unit(30px / @base-font-size-px, em);

    .respond-to-min(@bp-sm-min, {
      .grid_column(5);
      .grid_push(1);
      padding-top: 0;
      padding-bottom: 0;
      margin-top: 0;
    });
  }
```

Obviously totally ignore that if what you had better matches the comps :smile: